legate/driver/command.py

```diff
@@ -258,26 +259,37 @@ def cmd_openmp(
     )
 
 
-def cmd_utility(
+def cmd_bgwork(
     config: ConfigProtocol, system: System, launcher: Launcher
 ) -> CommandPart:
-    utility = config.core.utility
     ranks = config.multi_node.ranks
+    utility = config.core.utility
 
-    if utility == 1:
-        return ()
-
-    opts: CommandPart = ("-ll:util", str(utility))
+    opts: CommandPart = ()
 
     # If we are running multi-rank then make the number of active
     # message handler threads equal to our number of utility
     # processors in order to prevent head-of-line blocking
     if ranks > 1:
         opts += ("-ll:bgwork", str(utility))
```

Comment:
The default for `-ll:bgwork` is 2, so let's just always allocate at least 2.
```suggestion
        opts += ("-ll:bgwork", max(str(utility), 2))
```

Reply:
@SeyedMir this heuristic, which sets `-ll:bgwork` equal to `-ll:util`, was (presumably) appropriate for gasnet. Is there a better default for this setting when using the UCX backend?

Reply:
bgwok 2 has shown the best performance with the UCX backend in my tests. I did not test with higher `-ll:util` values.